CODE_OF_CONDUCT.md

```diff
@@ -1,13 +1,13 @@
 # Code of Conduct
 
-We are a community based on openness and friendly, didactic, discussions.
+We are a community based on openness, friendly, and didactic discussions.
```

Comment:
I agree the original version is a little hard to parse. In the original, I think "openness" was to describe the community. "Friendly" could either be describing the community or discussions. Maybe:

> We are an open community based on friendly and didactic discussions.

What do you think @avinash-ak ?

Reply:
Hi @thomasjpfan thanks for the review :), i agree. If it is fine could you please merge the PR.

Reply:
For updates to the code of conduct, I would prefer to wait for other core devs to review.